.github/workflows/build.yml

```diff
@@ -73,3 +73,27 @@ jobs:
         env:
           QUAY_USERNAME: ${{ secrets.QUAY_USERNAME }}
           QUAY_PASSWORD: ${{ secrets.QUAY_PASSWORD }}
+
+  publish_multi_platform:
+    name: Publish Multi-platform Images
+    # if: github.event_name == 'push' && github.ref == 'refs/heads/main' && github.repository == 'pypa/manylinux'
+    runs-on: ubuntu-latest
+    needs: build_manylinux
+    steps:
+      - name: Wait for Travis CI
+        uses: fountainhead/action-wait-for-check@v1.1.0
+        id: wait-for-travis
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          checkName: Travis CI - Pull Request
+          ref: ${{ github.event.pull_request.head.sha || github.sha }}
+          intervalSeconds: 10
+          timeoutSeconds: 3600
```

Comment:
Unfortunately, I expect the Travis CI check to fail every now and then (appart from the timeout being way too short for a Travis CI build).

This should really be a different workflow, possibly triggered by the checks being updated on main ?

Reply:
A different workflow should be fine, but it's harder to test since it can only run after merging. 

You could extract it to a separate workflow afterwards.

Reply:
It's fine to leave this here at the moment as you're saying.
It shouldn't be to hard to split with the deployment script being a separate bash script.